Makefile

```diff
@@ -88,12 +100,14 @@ install_python_sdk::
 
 install_go_sdk:: 
 
+install_java_sdk:: 
+
 install_nodejs_sdk:: 
 	yarn link --cwd $(WORKING_DIR)/sdk/nodejs/bin
 
-install_sdks:: install_dotnet_sdk install_python_sdk install_nodejs_sdk
+install_sdks:: install_dotnet_sdk install_python_sdk install_nodejs_sdk install_java_sdk
```

Comment:
if `install_java_sdk` isn't really doing anything, do we need to reference it here?

Maybe I'm missing something.